velox/functions/sparksql/tests/DecimalArithmeticTest.cpp

```diff
@@ -37,13 +37,15 @@ class DecimalArithmeticTest : public SparkFunctionBaseTest {
       const std::string& functionName,
       const VectorPtr& expected,
       const std::vector<VectorPtr>& inputs) {
-    VELOX_USER_CHECK_EQ(
+    VELOX_USER_CHECK_GE(
```

Comment:
Please check by the data type, then the fuzzer test can run to the correct path.

Reply:
Could you explain more about this comment. I may not get your point here.

Reply:
if (inputs[0]->tyle()->isDecimalType) {
check inputs.size == 1} else {check inputs.size == 2},
The fuzzer test will failed early when it generates 2 arguments for decimal type

Reply:
I think we can't differentiate by the `inputs[0]->type()`, before this function is for testing binary decimal arithmetic ops like Add(Decimal, Decimal). Now it also covers the test for fucntion `unaryminus(x: decimal(p, s))`.
This is test class, I'm not sure why it would impatt fuzzer test. does fuzzer test use this class?

Reply:
I missed this is the test file. It is OK now.